### PR TITLE
add Gateway instruction to readme

### PR DESCRIPTION
**Pull request by Abdiramen**
## What
Add install instructions for gateway preview

### CODE REVIEW

**Comment by Abdiramen** [comment]
* **#352** <a href="https://app.graphite.dev/github/pr/ngrok/kubernetes-ingress-controller/352?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Abdiramen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: README.md [hunk 1]
```diff
@@ -62,6 +65,25 @@ helm install ngrok-ingress-controller ngrok/kubernetes-ingress-controller \
 
 For a more in-depth installation guide follow our step-by-step [Getting Started](https://ngrok.com/docs/using-ngrok-with/k8s/) guide.
 
+#### Gateway API Prview
+
+To install the developer preview of the gateway api we'll make the following changes to the above instructions.
+
+Install the v1 gateway CRD before helm installing.
```
Comment (bobzilladev) [inline README.md:72]: Maybe `installation` instead?

**Review by bobzilladev — APPROVED** [review]
One small comment

File: README.md (hunk 1 above)
Comment (Abdiramen) [reply README.md]: updated here: https://github.com/ngrok/kubernetes-ingress-controller/pull/352/commits/bb95b457cc23c169f012fc6aac4d1e67a0f89452

**Review by bobzilladev — APPROVED** [review]

**Pull request merged**